### PR TITLE
Patch for the new variant consequence

### PR DESCRIPTION
**Pull request by olaaustine**
[G2P-101](https://www.ebi.ac.uk/panda/jira/browse/G2P-101)

### CODE REVIEW

File: sql/patch_110_111_b.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+-- Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+-- Copyright [2016-2024] EMBL-European Bioinformatics Institute
```
Comment (dglemos) [inline sql/patch_110_111_b.sql]: These first two lines should be removed.

File: sql/patch_110_111_b.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+-- 
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+
+
+#28-03-2024
```
Comment (dglemos) [inline sql/patch_110_111_b.sql]: Can also remove this.

**Review by dglemos — APPROVED** [review]

**Comment by dglemos** [comment]
Merged into release/111 and master

**Comment by olaaustine** [comment]
Patch added to the live DB

**Pull request merged**